src/test/resources/features/ErrorMessages.feature

```diff
@@ -17,7 +17,7 @@ Feature: Book a Visit Error messages
 
     Examples:
       | userName  | password          |
-      | VSIP1_TST | DigitalServices10 |
+      | VSIP5_TST | DigitalServices10 |
```

Comment:
Have we got a ticket to move these passwords out of GitHub (publicly available 😬!) and into something like a CircleCI environment variable?

Reply:
This is a known risk that's been addressed earlier Tom.